app/views/admin/merchants/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1 align = "center">Merchants</h1>
```

Comment:
There should not be spaces around the = in HTML. Don't need to fix now but we should address in refactor.